static/build-static

```diff
@@ -0,0 +1,391 @@
+#!/usr/bin/env bash
+set -e
+
+source "../scripts/target-platform"
+
+CURDIR="$1"
+TARGETPLATFORM="${2:-$CURPLATFORM}"
+
+if [ -z "$CURDIR" ] || [ -z "$TARGETPLATFORM" ]; then
+	# shellcheck disable=SC2016
+	echo 'usage: ./build-static ${CURDIR} ${TARGETPLATFORM}'
+	exit 1
+fi
+
+source "../scripts/target-platform"
```

Comment:
Yes it's on purpose to handle detected `TARGETPLATFORM` on first pass if `$2` is empty, otherwise takes user input and source it again to expand necessary vars. Will add a comment.